platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/NonNormalizedIdentityImmutableTransformExecution.java

```diff
@@ -28,12 +28,11 @@
 import java.io.File;
 import java.util.Map;
 
-class ImmutableTransformExecution extends AbstractTransformExecution {
+class NonNormalizedIdentityImmutableTransformExecution extends AbstractTransformExecution {
```

Comment:
🤔 Maybe we can skip "Identity" from the name and have only `NonNormalizedImmutableTransformExecution`

Reply:
Or maybe even `UnnormalizedImmutableTransformExecution`

Reply:
I talked to the Chat about this, here's what it said, and I mostly agree:

> "Non-normalized" is more neutral and simply states the absence of normalization without implying that normalization should have occurred. It suggests that the data has not been subjected to the process of normalization, and it's more of a passive state. It's commonly used in technical and scientific contexts where precision is valued.
> 
> "Unnormalized" could suggest that the data was normalized at some point and has been changed back to its original state, or it could also just indicate the lack of normalization. However, in practice, "unnormalized" is often used in mathematics and computing to simply mean "not normalized," without necessarily implying a process of denormalization.
> 
> Given that "normalized" data implies that an active process of normalization has been applied, using "non-normalized" would unambiguously indicate that this process has not been applied, and there is no suggestion that it is a state that has been reverted to. Thus, "non-normalized" might be the more technically appropriate term for your case, emphasizing that this is the default or natural state.

https://chat.openai.com/share/2901f5fc-81a4-4c6c-a3d3-604e19e1b90f